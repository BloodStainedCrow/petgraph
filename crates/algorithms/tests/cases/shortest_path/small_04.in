version https://git-lfs.github.com/spec/v1
oid sha256:850e20f1b0980ca4e7041b8c8c3d14d822c09d5af9f268530c99e7758f5e2576
size 868
