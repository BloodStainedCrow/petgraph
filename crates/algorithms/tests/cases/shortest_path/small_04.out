version https://git-lfs.github.com/spec/v1
oid sha256:1a14dbe2303ef65ec8df9e16d0a48d47fac2ed2828936af9c356e61f3e1fe8c2
size 40
